tests/php/Chat/Parser/SystemMessageTest.php

```diff
@@ -611,7 +618,7 @@ public function testGetFileFromShareForGuest() {
 		$node = $this->createMock(Node::class);
 		$node->expects($this->once())
 			->method('getId')
-			->willReturn('54');
+			->willReturn(54);
```

Comment:
Just to spill over from chat, this looks fishy and needs to be confirmed.
API docs say int, but it's alerting that string was tested before. Maybe reality of docs and API result diverge

Reply:
If we need to revert that, we need to cast to `int` for the metadata API, as this API requires a `int`.

Reply:
> its int it seems
> that's also why we cast it to string on the rich object